core/arch/arm/plat-sam/pm/psci.c

```diff
@@ -3,16 +3,39 @@
  * Copyright (c) 2021, Microchip
  */
 
+#include <drivers/atmel_rstc.h>
```

Comment:
In commit "plat-sam: enable PSCI reset support": `console.h` should be before `drivers/xxx.h`.

Reply:
Even better, removed it since not needed anymore.